stories/message-page/message-page.stories.js

```diff
@@ -0,0 +1,212 @@
+- The title is also mandatory. Add a description with further details if needed. 
+- The message should be as concise as possible.
+        `,
+        tags: ['f3', 'a11y', 'theme', 'development']
+    }
+};
+
+const messageBoxHeight = 800;
+
+export const filter = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--filter fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No matching items found.
+                </div>
+                <div class="fd-message-page__subtitle">
+                    Check the filter settings.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+filter.storyName = 'Message Page with Filter';
+filter.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The user has set a filter and there are no results.
+
+Icon: <code class="docs-code">.sap-icon--filter</code>
+    `
+    }
+};
+
+export const search = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--search fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No matching items found.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+search.storyName = 'Message Page with Search';
+search.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The user has searched for something but there are no results.
+
+Icon: <code class="docs-code">.sap-icon--search</code>
+    `
+    }
+};
+
+export const noItems = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--product fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No items are currently available.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+noItems.storyName = 'Message Page with No Items';
+noItems.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The app contains no items.
```

Comment:
"Use the no items message page for when the user has viewed a page that is supposed to contain items, but there are none.

Mandatory: Display a **product icon**, or any other icon that matches your use case."

Reply:
I bolded product icon because I assume that this is a known icon name in Fiori? Can you confirm?

Reply:
Otherwise don't bold it in the documentation!

Reply:
Done